.vscode/settings.json

```diff
@@ -41,7 +41,6 @@
     "editor.formatOnSave": true
   },
 
-  "python.linting.pylintEnabled": false,
```

Comment:
Does this default to false ?

Reply:
I'm not sure, but there's no sense in having pylint at all since flake8 is being used and enforced in CI.